tests/integration.rs

```diff
@@ -4073,8 +4073,8 @@ mod common_parallel {
     fn test_serial_socket_interaction() {
         let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
         let guest = Guest::new(Box::new(focal));
-        let serial_socket = guest.tmp_dir.as_path().join("/tmp/serial.socket");
-        let serial_socket_pty = guest.tmp_dir.as_path().join("/tmp/serial.pty");
+        let serial_socket = guest.tmp_dir.as_path().join("tmp/serial.socket");
```

Comment:
Did you have these changes in for testing? They broke the CI.

Reply:
Sorry I didn't notice the problem.
I checked the source again, it works now.